### PR TITLE
fix the use of DISTCHECK_CONFIGURE_FLAGS

### PR DESCRIPTION
**Pull request by daveol**
now both ```./configure``` and ```make distcheck``` use the same flags

### CODE REVIEW

**Comment by nomis52** [comment]
LGTM

File: .travis-ci.sh [hunk 1]
```diff
@@ -74,10 +74,10 @@ elif [[ $TASK = 'jshint' ]]; then
   npm install;
   grunt test
 else
-  # Otherwise compile and check as normal
```
Comment (peternewman) [inline .travis-ci.sh]: Can we keep the comment please.

**Comment by peternewman** [comment]
LGTM

**Comment by daveol** [comment]
do we want to wait for travis to go green? or do we merge now?

**Comment by nomis52** [comment]
Can you merge and try the tests again, they should work now.

**Comment by peternewman** [comment]
They won't work until https://github.com/OpenLightingProject/ola/pull/843 goes in, which is as soon as it goes green.

**Comment by peternewman** [comment]
#843 is in now if you resync @daveol

**Comment by peternewman** [comment]
Still LGTM

**Pull request merged**